owncloudApp/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.kt

```diff
@@ -516,11 +513,7 @@ class FileDisplayActivity : FileActivity(), FileFragment.ContainerActivity, OnEn
      */
     override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
 
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
-            BiometricManager.getBiometricManager(this).bayPassUnlockOnce()
-        }
-        PassCodeManager.bayPassUnlockOnce()
-        PatternManager.getPatternManager().bayPassUnlockOnce()
+        bayPassUnlockOnce()
```

Comment:
This is a change I didn't plan but with the change of the logic of `bayPassUnlockOnce` and knowing now it's the same function, I think we can save all these lines! I tested it and it seems to work 😄